config/sync/auto_entitylabel.settings.node.stanford_person.yml

```diff
@@ -1,6 +1,6 @@
 status: 1
 pattern: '[node:su_person_first_name] [node:su_person_last_name]'
-escape: true
+escape: false
```

Comment:
What are the security implications of this change?

Reply:
No different than putting special characters in the title on other content types. shouldn't be any security issue.

Reply:
this is also the same settings we currently have for news content types.

Reply:
So Drupal sanitizes it prior to putting it in the DB? Can you add a [Little Bobby Tables](https://xkcd.com/327/) test?

Reply:
Drupal doesn't often sanitize writing to the db, but rather, wants you to sanitize on output.

Reply:
That was my recollection, but I did not phrase my question articulately. Is it safe to say that the DB abstraction layer protects against this not being escaped?